shared/src/test/scala/io/circe/jackson/CirceSuite.scala

```diff
@@ -4,25 +4,13 @@ import cats.instances.AllInstances
 import cats.syntax.{ AllSyntax, EitherOps }
 import io.circe.Json
 import io.circe.testing.{ ArbitraryInstances, EqInstances }
-import org.scalatest.flatspec.AnyFlatSpec
-import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
-import org.typelevel.discipline.scalatest.FlatSpecDiscipline
+import munit.ScalaCheckSuite
 import scala.language.implicitConversions
 
 /**
  * An opinionated stack of traits to improve consistency and reduce boilerplate in circe tests.
  */
-trait CirceSuite
-    extends AnyFlatSpec
-    with FlatSpecDiscipline
-    with ScalaCheckDrivenPropertyChecks
-    with AllInstances
-    with AllSyntax
-    with ArbitraryInstances
-    with EqInstances {
-
-  override def convertToEqualizer[T](left: T): Equalizer[T] =
-    sys.error("Intentionally ambiguous implicit for Equalizer")
+trait CirceSuite extends ScalaCheckSuite with AllInstances with AllSyntax with ArbitraryInstances with EqInstances {
 
   implicit def prioritizedCatsSyntaxEither[A, B](eab: Either[A, B]): EitherOps[A, B] = new EitherOps(eab)
```

Comment:
I don't think we need this now, do we?

Reply:
No. Dropped.

Reply:
:+1: Incidentally I think we can get rid of `AllInstances` now, too, thanks to Cats 2.2.